.github/workflows/tests.yml

```diff
@@ -0,0 +1,43 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '12.x', '14.x', '16.x' ]
```

Comment:
We should probably test on 10 since we still support it and not supporting would be a breaking change.

Reply:
added it back